packages/plugin-list/test/List.test.ts

```diff
@@ -111,7 +111,7 @@ describePlugin(List, testEditor => {
                 '        <li>b.4</li>', // li3_3
                 '    </ol>',
                 '    <ul class="checklist">', // li4
-                '        <li class="unchecked">c.1</li>', // li4_0
+                '        <li>c.1</li>', // li4_0
```

Comment:
I thought chm went back on his "you can delete it" statement regarding `unchecked` ?